neo/Consensus/ConsensusService.cs

```diff
@@ -70,13 +70,12 @@ private bool AddTransaction(Transaction tx, bool verify)
             {
                 if (context.VerifyRequest())
                 {
-                    context.State |= ConsensusState.ResponseSent;
```

Comment:
It needs to do something to prevent it from going to the wrong code path in `OnTimer` after receiving the `RecoveryMessage` when accepting it's own prepare request by a `RecoveryMessage`. This was one way to achieve it.

Reply:
Which wrong code path in `OnTimer()`?